<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadingPage</class>
 <widget class="QWidget" name="LoadingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advance Automatic Attendance System</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Images/applogo.png</normaloff>../Images/applogo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(6, 40, 61);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,255,255);
font: 75 20pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Advance Automatic Attendance System</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>621</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Images/wallpaper.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>621</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="startBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(240, 240, 240);
border-radius:10px;
}</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Images/images.qrc"/>
 </resources>
 <connections/>
</ui>
